packages/@glimmer/application/src/loaders/bytecode/resolver.ts

```diff
@@ -65,6 +65,11 @@ export default class BytecodeResolver implements RuntimeResolver<TemplateMeta> {
     }
 
     let trimmed = templateSpecifier.replace(this.prefix, '');
+
+    if (!this.meta[trimmed]) {
+      throw new Error(`Could not find ${trimmed} in metadata map for bytecode. Keys that exist in the map: ${Object.keys(this.meta)}`);
```

Comment:
Awesome work! I'm wondering if this error message is too specific to the internal implementation.

Maybe something like `Could not find component <${componentName}> invoked from the <${referrerComponentName>. Available components are: ${Object.keys(this.meta)}`? I think this might better match the mental model of the person who encounters the error.

Reply:
Yup makes more sense. Fixed it up.